modules/src/main/modules/search/elasticsearch/module.xml

```diff
@@ -8,17 +8,13 @@
 <module xmlns="urn:jboss:module:1.1" name="org.hibernate.search.elasticsearch" slot="${hibernate.search.version}">
     <resources>
         <resource-root path="hibernate-search-elasticsearch-${hibernate.search.version}.jar" />
-        <resource-root path="rest-${elasticsearchClientVersion}.jar" />
-        <resource-root path="sniffer-${elasticsearchClientVersion}.jar" />
-        <resource-root path="gson-${elasticsearchGsonVersion}.jar" />
```

Comment:
Hum, ok. I thought this was about laying the groundwork to contribute a Wildfly module to Elastic, but it seems I was confused.